website/docs/r/wafregional_web_acl_association.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: "aws"
+page_title: "AWS: aws_wafregional_web_acl_association"
+sidebar_current: "docs-aws-resource-wafregional-web-acl-association"
+description: |-
+  Provides a resource to create an association between a WAF Regional WebACL and Application Load Balancer.
+---
+
+# aws\_wafregional\_web\_acl\_association
+
+Provides a resource to create an association between a WAF Regional WebACL and Application Load Balancer.
+
+-> **Note:** An Application Load Balancer can only be associated with one WAF Regional WebACL.
+
+## Example Usage
+
+```hcl
+resource "aws_wafregional_ipset" "ipset" {
+  name = "tfIPSet"
+  
+  ip_set_descriptors {
+    type  = "IPV4"
+    value = "192.0.7.0/24"
+  }
+}
+
+resource "aws_wafregional_rule" "wafrule" {
+  depends_on  = ["aws_wafregional_ipset.ipset"]
+  name        = "tfWAFRule"
+  metric_name = "tfWAFRule"
+  
+  predicates {
+    data_id = "${aws_wafregional_ipset.ipset.id}"
+    negated = false
+    type    = "IPMatch"
+  }
+}
+
+resource "aws_wafregional_web_acl" "wafacl" {
+  depends_on  = ["aws_wafregional_ipset.ipset", "aws_wafregional_rule.wafrule"]
```

Comment:
The `depends_on` is redundant here as the relationship is already defined by the references below. Do you mind removing it?